appveyor.yml

```diff
@@ -26,12 +24,6 @@ for:
   - ps: .\build.ps1
   - pwsh: .\build.ps1
 
-- matrix:
-    only:
-      - image: Ubuntu
-  environment:
-    LANG: us_US.UTF-8
-
 - matrix:

```

Comment:
Make this `matrix` block look like [this](https://github.com/webmd-health-services/Carbon.Cryptography/blob/main/appveyor.yml#L27-L49).